presidio-analyzer/tests/test_analyzer_engine.py

```diff
@@ -265,3 +285,73 @@ def test_when_allFields_is_true_and_entities_not_empty_exception(self):
         new_field.minScore = '0.5'
         with pytest.raises(ValueError):
             analyze_engine.Apply(request, None)
+
+    # Context tests
+    def test_text_with_context_improves_score(self):
+        test_items = self.load_test_items()
+
+        # Currently we have 19 sentences, this is a sanity
+        assert(len(test_items) == 19)
```

Comment:
This assert tests no business logic and should be removed.
In general Asserts should appear at the end of the test, after the "act" part of the UT.

Reply:
The asserts allow us to gain confidence.
in this case I made sure if someone empties the text file it will be noticeable.
without this assert you can easily lose UT coverage

Reply:
I disagree. UT coverage - is for business logic. 
`self.load_test_items()` is a code written in a test file, not in the application.
Why would you test your test using assertion? 
Validating your Test Data should be done in the create function `load_test_items`. If you fail to create the data well, there is no point in running any test afterwards anyway. 
BTW, why is it so important to to have 19 items?  what is sanity about it?

Reply:
as long as we agree it should be checked. I have moved it into the 'load' method.
as I wrote before, this test has 19 examples, and its a good idea to make sure this file is intact.